yb-voyager/src/callhome/diagnostics.go

```diff
@@ -112,6 +112,7 @@ type AssessMigrationPhasePayload struct {
 	IndexSizingStats               string                    `json:"index_sizing_stats"`
 	SourceConnectivity             bool                      `json:"source_connectivity"`
 	IopsInterval                   int64                     `json:"iops_interval"`
+	YugabyteDUIEnabled             bool                      `json:"yugabyted_ui_enabled"`
```

Comment:
Shall we just send "control_plane_type" instead of a boolean? 
That will cover for other control plane types as well in the future.

Reply:
I think sending the direct information helps; else we have to keep track of the values of control_plane_type in callhome queries, and in the future, if we change it we will have to update that.

Reply:
> Shall we just send "control_plane_type"

I think that might be more helpful and future-proof.
In anycase we will have to check for the flag's value

We can have this as fixed const var to incorporate multiple values - `yugabyted`, `yugabytedb_anywhere`, `yugabytedb_aeon`

If we are concerned about case - voyager sending to multiple control plane at one time then maybe use `[]string`

Reply:
Yeah, I still feel like `control_plane_type` would be better. 

>we have to keep track of the values of control_plane_type in callhome queries, and in the future, if we change it we will have to update that. 

The way I look at it is: Send all the raw data to callhome, and write queries at that layer for interpretation (derived data). 
So, in this context, the control_plane_type is the raw data. "Whether yugabytedUI was used or not" is a derived data point, which can be defined at the callhome layer. 

If/when we introduce new control plane tyeps (yba,ybaeon), then there will be no change required in voyager, only in the callhome layer, which seems ideal, don't you think?

Reply:
Hmm, okay, I got your point. Let me make that change.

Reply:
Is it okay to assume a single control plane type for now?

Reply:
I am really not sure about this - how does it work in real world usecase.
Depends on if it is normal or frequent for a tool or service to be managed by multiple control planes.

cc @makalaaneesh

Reply:
I think assumption of a single control plane is fine for now. Typically, that is how it is going to be set up. Don't see why one would need multiple control planes. 
I don't think our code allows multiple control planes either..